### ISSUE TITLE
3 out of 4 published videos won't play

### ISSUE THREAD

**Issue by ITwrx** [lbryio/lbry-desktop#2305]
## The Issue
3 out of 4 published videos won't play. either they weren't really successfully uploaded or something is preventing playing them. they appear as if they are there and ready to play but they don't play and the timeline dealy doesn't ever move nor does the counter start. no errors in the journal (system log) and lbry reported that all the videos uploaded fine and were ready to be viewed when i tried to watch them. i have edited them several times in case it was a failed upload but the same three videos still won't play.
### Steps to Reproduce

1. upload webm videos
2. try to watch them to make sure all is well
3. soak in the fail.

### Expected Behavior
they should play.

### Actual Behavior
only 1 out of the 4 vids will play. i notice that the one that works is much longer than the others. all the ones that won't play are under 3 minutes. maybe there's a minimum length for some reason?

### Suggested Solutions
i don't know the cause so i don't have a solution but it would be nice to get an error message when either the upload is not truly successful or anytime a video won't play. if there is a minimum video length it would be nice to be told that in the upload/create view. I would rather not have this theoretical minimum, as i made these three videos short on purpose...

## System Configuration

- LBRY Daemon version:
{
  "jsonrpc": "2.0",
  "result": {
    "build": "release",
    "desktop": "GNOME",
    "distro": {
      "codename": "",
      "id": "arch",
      "like": "",
      "version": "",
      "version_parts": {
        "build_number": "",
        "major": "",
        "minor": ""
      }
    },
    "lbrynet_version": "0.32.4",
    "lbryschema_version": "0.0.16",
    "os_release": "4.20.13.a-1-hardened",
    "os_system": "Linux",
    "platform": "Linux-4.20.13.a-1-hardened-x86_64-with-arch",
    "processor": "",
    "python_version": "3.7.1"
  }
}
- LBRY App version: 0.29.2
- LBRY Installation ID: adding this feels like a potential long term privacy problem. 
- Operating system: arch linux

------------
thanks

**Comment by tzarebczan** [comment]
Hey, thanks for reaching out. What format are the videos in? Our in app video player only supports h264 mp4s currently, but any file can also be opened externally. We have a few other tickets outstand for video player issues similar to this, and the solution is to upgrade our video player. We are waiting on an SDK enhancement to do so, but it should be coming in the next couple of months.


Please email hello@lbry.io with the additional information about the videos you tried uploading.

**Comment by tzarebczan** [comment]
Closing in favor of https://github.com/lbryio/lbry-desktop/issues/1955

**Comment by ITwrx** [comment]
it was mentioned in the "steps to reproduce" 1, but no big deal. they are webm files and one of them plays. maybe non-h264 just doesn't play reliably?

**Comment by tzarebczan** [comment]
Webm is just a container like mp4 is. Non h264 videos will not play back in the app until we upgrade the video player.

Shoot us an email with your LBC address so we can show you some appreciation for taking the time to test out LBRY and report issues :)

**Comment by ITwrx** [comment]
i was afraid you might say that. i never can be bothered to remember which is the containers, etc. I just checked and all four videos are vp8 with vorbis audio in webm container. all recorded with simple screen recorder. 
thanks, i'll do that. :)

**Comment by tzarebczan** [comment]
Sounds good, and I'm also really looking forward to being able to support more formats and codecs.

**Comment by ITwrx** [comment]
btw, i scanned the FAQ prior to uploading my videos and it says 

> LBRY accepts any HTML5 format for video streaming; the full list can be found here. This means a web-optimized MP4 file is the best format.

this does not mean that mp4 is the best format for streaming...  The FAQ needs to be clearer about the facts, not just state opinions based on what might easier for people with crippled software. Webm is the best format for streaming, out of the two, if we are giving opinions. They are both supported by html5 video and webm is a Free codec and wrapper. 

this is why i used vp8 and webm to begin with. html5 video is what i use to serve these videos on my own site right now. The fact that one plays fine in lbry shows that this is a legit bug with the current implementation (of something), imho. 

now if lbry doesn't want to bother fixing it b/c you are working to change the player anyways that's understandable, but that will cause breakage and frustration for some people until then.

**Comment by seanyesmunt** [comment]
hey @ITwrx sorry about the issues. A new video player is one of our top priorities. I agree it's a huge pain not being able to stream some videos.

The upcoming lbry.tv, which is just the app code in the browser, will use this new player. Should be able to play them there.
